trillian/ctfe/ct_server/main.go

```diff
@@ -192,6 +192,12 @@ func main() {
 		}
 	})
 
+	// Export a healthz target.
+	corsMux.HandleFunc("/healthz", func(resp http.ResponseWriter, req *http.Request) {
+		// TODO(al): Wire this up to tell the truth.
```

Comment:
Maybe create an Issue to track this, unless you're planning on doing it imminently?

Reply:
I'll leave the current issue open and add a note there.